dom/src/MainDOMSource.ts

```diff
@@ -215,3 +265,11 @@ export class MainDOMSource implements DOMSource {
   public isolateSource: (source: DOMSource, scope: string) => DOMSource = isolateSource;
   public isolateSink: (sink: Stream<VNode>, scope: string) => Stream<VNode> = isolateSink;
 }
+
+export interface MainDOMSourceOptions extends SourceOptions {
```

Comment:
Only if you don’t want the code to read like a book. The `x - n` rule makes the reader scroll up and down, up and down. This is especially true for functions.

Personally, I like to open a file and immediately see what it exports without the need to scroll 100 or more lines down. But if you insist.